index.html

```diff
@@ -364,7 +364,8 @@ <h4>Surprising facts about the Monstera</h4>
             </p>
 
           </div> 
-          <button><a href="signup.html">Read more</a></button>
+          <button><a id="change-tect-color" href="signup.html">Read more</a></button>

```

Comment:
change id name. typo error found (change-text-color instead of change-tect-color)